docs/conformance.md

```diff
@@ -1,83 +1,299 @@
+  characterization details uniquely specialized to their particular context.
+  Enables adopting individuals or communities to utilize SPDX expressive
+  capabilities along with expressing more arcane Element characterization
+  details specific to them and not appropriate for standardization across SPDX.
+
+- Support structured capture of expressive solutions for gaps in SPDX coverage
+  from real-world use. Enables adopting individuals or communities to express
+  Element characterization details they require that are not currently defined
+  in SPDX but likely should be. Enables a practical pipeline that identifies
+  gaps in SPDX that should be filled, expresses solutions to those gaps in a
+  way that allows the identifying adopters to use the extended solutions with
+  SPDX and does not conflict with current SPDX, can be clearly detected among
+  the SPDX content exchange ecosystem, provides a clear and structured
+  definition of gap solution that can be used as submission for revision to the
+  SPDX standard.
+
+Software that conforms to the SPDX specification at the Extension Profile
+compliance point shall be able to import and export serialized documents that
+conform with one of the SPDX serialization formats defined SPDX serialization
+formats, including the abstract Extension class serving as the base for all
+defined Extension subclasses.
+
+Conformance to the Extension Profile compliance point does not entail support
+for the Licencing, Security, Dataset, AI, Build, or profiles of the SPDX but
+is expected to be used in combination with the other profiles to extend them.
+
+This compliance point facilitates interchange of extended information that goes
+beyond the standard SPDX produced by tools supporting SPDX and is used between
+cooperating parties that understand the form of the extension and can produce
+and consume its non-standard content.
+
+## 5.12 Standard data format requirements <a name="5.12"></a>
+
+The data format specification and recommendations are subject to the
+following constraints:
+
+- Shall be in a human readable form.
+- Shall be in a syntax that a software tool can read and write.
+- Shall be suitable to be checked for syntactic correctness automatically,
+  independent of how it was generated (human or tool).
+- The SPDX document character set shall support UTF-8 encoding.
+- Multiple serialization formats may be used to represent the information being
+  exchanged. Current supported formats include:
+
+  - **YAML 1.2**
+    see:
+    [YAML Ain’t Markup Language (YAML™) version 1.2](https://yaml.org/spec/1.2/)
+  - **JavaScript Object Notation** (JSON)
+    see:
+    [ECMA-404](https://ecma-international.org/publications-and-standards/standards/ecma-404/)
+    - The JSON Schema for SPDX can be found in the
+      [SPDX specification Git repository schemas directory](https://github.com/spdx/spdx-spec/blob/master/schemas/spdx-schema.json)
```

Comment:
This URL is not exist

Reply:
@bact - good catch.

Reply:
TO Discuss:  should we merge, then update?  or wait for fix here.

Reply:
If we can find it quickly today, maybe wait.
If not, we can have another PR to fix that.